plugins/OAuth2/views/settings.php

```diff
@@ -0,0 +1,25 @@
+<h1><?php echo $this->data('Title'); ?></h1>
+
+<div class="PageInfo">
+    <h2><?php echo t('Create Single Sign On integration using the OAuth2 protocol!'); ?></h2>
+
+    <p>
+        <?php
+        echo t('<p>If you haven\'t already, create an SSO application.</p>
```

Comment:
This should have a slug of something like `OAuth2 setup instructions text` and likewise be added to the locales repo in the file for longer text strings.